pyaptamer/datasets/_loaders/load_aptamer_interactions.py

```diff
@@ -0,0 +1,235 @@
+__author__ = "Satarupa22-SD"
+__all__ = ["load_aptadb", "load_aptamer_interactions", "load_interactions"]
+
+from pathlib import Path
+
+import pandas as pd
+
+
+def _download_dataset(
+    dataset_name: str, target_dir: Path, force_download: bool = False
+) -> None:
+    """Download a Kaggle dataset to the specified directory and unzip it.
+
+    This is a private helper function used internally by the module.
+
+    Parameters
+    ----------
+    dataset_name : str
+        The Kaggle dataset identifier in format "username/dataset-name"
+    target_dir : Path
+        Directory where the dataset should be downloaded and extracted
+    force_download : bool, default False
```

Comment:
Replace with [`overwrite`](https://github.com/gc-os-ai/pyaptamer/blob/main/pyaptamer/datasets/_loaders/_online_databank.py)

Reply:
is overwrite really better? It odes not always overwrite, e.g., if it is the first time using the function. I would leave `force_download`, this is also how HF calls the arg
https://huggingface.co/docs/huggingface_hub/en/package_reference/file_download